lib/safer_rails_console/patches/boot/sandbox_flag.rb

```diff
@@ -55,6 +57,8 @@ def parse_arguments(arguments)
               OptionParser.new do |opt|
                 opt.banner = 'Usage: rails console [environment] [options]'
                 opt.on('-s', '--[no-]sandbox', 'Explicitly enable/disable sandbox mode.') { |v| options[:sandbox] = v }
+                opt.on('-w', '--writable', 'Alias for --no-sandbox.') { |v| options[:writable] = v }
```

Comment:
How hard is it to share this code with `Console4` code above?

Reply:
Fairly difficult - I'm redefining `parse_arguments`, where the rest of the method outside of the `OptionParser.new` block is different

Reply:
Could we use a concern with a method `configure_option_parser` and just call that in both places?  Not sure if we expect this code to always be the same or diverge at some point.

Reply:
Discussed offline - will implement if it makes sense given slightly divergent code between Rails 4.x and 5.0